benchmarks/benchmarks/solarposition.py

```diff
@@ -43,13 +41,66 @@ def time_spa_python(self, ndays):
     def time_pyephem(self, ndays):
         solarposition.pyephem(self.times_localized, self.lat, self.lon)
 
+    def time_nrel_earthsun_distance(self, ndays):
+        solarposition.nrel_earthsun_distance(self.times_localized)
+
+    def time_pyephem_earthsun_distance(self, ndays):
+        solarposition.pyephem_earthsun_distance(self.times_localized)
+
+    def time_get_solarposition(self, ndays):
+        solarposition.get_solarposition(self.times_localized,
+                                        self.lat, self.lon)
```

Comment:
I'm not sure there's any need to benchmark `get_solarposition` -- since it's just a convenience wrapper around the various solar position algorithms, this is more or less a duplicate of the `time_spa_python` benchmark.